### PR TITLE
Remove api key from mandatory config

### PR DESCRIPTION
**Pull request by bidoubiwa**

### CODE REVIEW

**Review by alallema — DISMISSED** [review]
LGTM!

**Review by curquiza — COMMENTED** [review]
Could we update the README to say `apiKey` is optional? :)

File: README.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ If you already have your own scraper but you still want to use MeiliSearch and `
 </html>
 ```
 
-The `hostUrl` and the `apiKey` fields are the credentials of your MeiliSearch instance.<br>
+The `hostUrl` and the `apiKey` (_optionnal_) fields are the credentials of your MeiliSearch instance.<br>
```
Comment (curquiza) [inline README.md:94]:
```suggestion
The `hostUrl` and the `apiKey` (_optional_) fields are the credentials of your MeiliSearch instance.<br>
```

**Comment by bidoubiwa** [comment]
bors merge

**Review by curquiza — APPROVED** [review]

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**